ManagementPacks/Community.DataOnDemand/Modules/WatchTcpConnectionsWA.mpx

```diff
@@ -0,0 +1,68 @@
+﻿<ManagementPackFragment SchemaVersion="2.0" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+
+  <TypeDefinitions>
+    <ModuleTypes>
+      <WriteActionModuleType ID="Community.DataOnDemand.WriteAction.WatchTcpConnections" Accessibility="Public" Batching="false">
```

Comment:
This shouldn't be a WriteAction - it's not causing any changes.  Please convert this into a Probe (the original task modules were created as WriteActions erroneously so it's too late for them - see the ResolveAddress probe as an example).